src/_assets/css/_base.scss

```diff
@@ -9,6 +9,7 @@ h1 {
 
 h2 {
   margin: bs-spacer(12) 0 bs-spacer(8);
+  clear: both;
```

Comment:
Hmm, I'd prefer to not apply this to h2 generally. I think that in individual cases, we can add use of the BS `clearfix` class in the markup/markdown file. WDYT?

Reply:
I didn't do it for h3s and below because it's possible an image would be floating in that section. We can certainly use 'clearfix' we just have to educate the writers that whenever they use a 'site-image-right` they should add a clearfix to the next section.

Reply:
> The problem is that if we do it as a class we would need to add it every time there is a floating image.

I don't expect that it will happen that often.

> I thought adding to the h2 top level would be ok since when using an h2 in the docs we are indicating a new high level section, and it's very unlikely the h2 should continue floating next to an image in the previous section.

Ok, let's try it this way for now.